package.json

```diff
@@ -24,8 +24,4 @@
     "imagemin-mozjpeg": "^8.0.0",
     "imagemin-pngquant": "^8.0.0",
     "imagemin-webp": "^5.1.0"
-  },
-  "dependencies": {
-    "http-server": "^0.12.3"
-  }
```

Comment:
You need to run `npm install` to remove the dependency from the `package-lock.json` file. Otherwise it is still technically a dependency of the project.